README.md

```diff
@@ -35,6 +35,12 @@ A list of every a11y concern Checka11y.css will check for and highlight:
 
 - Missing `alt` attributes on images
 - Missing `title` on `<iframe>`
+<<<<<<< HEAD
```

Comment:
Please revert all changes in this file and ensure it is up to date with `master`. 😊